llvm/lib/MC/XCOFFObjectWriter.cpp

```diff
@@ -649,6 +649,14 @@ void XCOFFObjectWriter::recordRelocation(MCAssembler &Asm,
                                          uint64_t &FixedValue) {
   auto getIndex = [this](const MCSymbol *Sym,
                          const MCSectionXCOFF *ContainingCsect) {
+    // Fixup relocation flag for AIX TLS local-dynamic mode.
```

Comment:
I will look into alternative approach to handle this fixup.

Reply:
Proposed to fix the setting for the "_$TLSML" symbol in `TargetLoweringObjectFileXCOFF::getSectionForExternalReference()`.